client/src/js/services/receipts/modal/receiptModal.tmpl.html

```diff
@@ -25,23 +25,22 @@
   <p class="text-info"><span class="glyphicon glyphicon-hourglass"></span> {{"RECEIPTS.LOADING"}}</p>
 </div>
 
-<div ng-switch on="ReceiptCtrl.renderer">
+<!-- ng-if used in favour of ng-switch to allow dynamic states -->
+<div>
 
   <!-- PDF -->
-  <div ng-switch-when="ReceiptCtrl.renderers.PDF">
+  <div ng-if="ReceiptCtrl.renderer === ReceiptCtrl.renderers.PDF">
 
     <iframe
       id="pdf"
       ng-src="{{ReceiptCtrl.receipt}}"
       type="application/pdf"
       style="width : 100%; height : 50vh; margin-bottom : -5px; border : none;">
-      <!-- TODO: Translate -->
-       <!--<p>Unable to display PDF. Please contact your system administrator</p>-->
     </iframe>
   </div>
 
   <!-- HTML -->
-  <div ng-switch-when="ReceiptCtrl.renderers.HTML">
+  <div ng-if="ReceiptCtrl.renderer === ReceiptCtrl.renderers.HTML">
```

Comment:
It's a shame this doesn't work as expected.  If we really needed to, we could write out own version of `ngSwitch`.  Using `ngIf` here is perfect though.